dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganizationResource.java

```diff
@@ -409,6 +410,31 @@ private Organization getOrganizationByIdOptionalAuth(Optional<User> user, Long o
         }
     }
 
+    @DELETE
+    @Path("/{organizationId}/deleteNonApprovedOrg")
```

Comment:
Path should simply be `/{organizationId}`. The `@DELETE` already says what it is doing to the org. In REST, think of paths as resources, not actions. The resource is the organization, specified by the id.

Reply:
Done. Changed the path to be simply /{organizationId}